sql/parser/builtins.go

```diff
@@ -470,3 +545,13 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+func nowBuiltin() builtin {
```

Comment:
It can be a var but then it's the only var here, so I felt it best matched everything else.